_faq.Rmd

```diff
@@ -237,63 +260,47 @@ reason is that these phonological descriptions represent different doculects
 with the same group of speakers, where "group of speakers" is ultimately
-determined by your desired level of granularity. To give a concrete example, one
-researcher may be interested in comparing "languages", and so might wish to
-treat all inventories of "English" as duplicates in some sense, regardless of
-the regional dialect, sociolect, etc., represented in the doculect and encoded in
-the phonological inventory. Such a researcher might *filter* or *sample* the
-PHOIBLE dataset in order to include only one inventory for each isocode (how she
-might choose to implement that filter is a separate question; see "Filtering and
-sampling inventories" for examples). Other researchers may not care about
+determined by your desired level of granularity. To give a concrete example,
+one researcher may be interested in comparing lects at the "language" level,
+and so might wish to treat all inventories of "English" as duplicates for the
+purposes of her analysis (regardless of any differences in regional dialect or
+sociolect represented in the original doculects and encoded in the phonological
+inventory). That researcher might *filter* or *sample* PHOIBLE's inventories to
+include only one inventory for each isocode (how she chooses to implement that
+filter is a separate question; see "How can I filter or sample inventories?"
+for examples). Other researchers may not care about
 "duplicates" in that sense, and may choose to include all inventories in their
 analysis (or, they may filter the dataset to include only inventories with a
 particular feature of interest such as breathy-voiced vowels).
 
-Because a key principle of PHOIBLE is to remain as faithful as possible to the description of speech sounds in the original doculects (though not necessarily to the exact *symbols* used in the document, if they don't reflect the textual descriptions; see for example our [IPA conventions](http://phoible.github.io/conventions/)), it may arise that linguists disagree on their phonological analysis of a particular languoid, and hence there may be two PHOIBLE inventories (with unique inventory IDs) that link to the same glottocode (but different documents), but nonetheless contain different sets of phonemes.
-
-For example, [Barbareño](https://en.wikipedia.org/wiki/Barbare%C3%B1o_language) has two inventories from different doculects that have slightly different phoneme inventories:
-
-* https://phoible.org/inventories/view/1239
-* https://phoible.org/inventories/view/862
-
-Inventory (PH) 1239 has 44 phonemes:
-
-```{r}
-phoible %>% filter(InventoryID == 1239) %>% pull(Phoneme) %>% cat("\n")
-```
-
-Inventory (PH) 862 has 40 phonemes:
-
-```{r}
-phoible %>% filter(InventoryID == 862) %>% pull(Phoneme) %>% cat("\n")
-```
-
-The difference between these two inventories is reflected clearly in their sources, i.e. [@Beeler1970] posits four additional segments (`sʼ ʃʰ ʃʼ χʼ`) whereas [@Wash2001] does not.
```

Comment:
I cut this because I think it doesn't really show anything different from the Kabardian example.